Sources/CGFloatExtensions.swift

```diff
@@ -62,4 +62,22 @@ extension CGFloat {
     public static func random(within: ClosedRange<CGFloat>) -> CGFloat {
         return CGFloat.random() * (within.upperBound - within.lowerBound) + within.lowerBound
     }
+    
+    /**
+      EZSE :Returns the shortest angle between two angles. The result is always between
+      -π and π.
+     
+      Inspired from : https://github.com/raywenderlich/SKTUtils/blob/master/SKTUtils/CGFloat%2BExtensions.swift
+     */
+    public static func shortestAngleBetweenInRadians(angle1InRadians: CGFloat, angle2InRadians: CGFloat) -> CGFloat {
```

Comment:
CGFloat.shortestAngleInRadians(_ angle1: firstAngle, _ angle2: fullCircleAngle)
CGFloat.shortestAngleInRadians(from angle1: firstAngle, to angle2: fullCircleAngle)
?